src/app/shared/components/favorite-toggle/favorite-toggle.component.ts

```diff
@@ -1,17 +1,20 @@
 import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { NotificationService } from 'src/core/api/notification/notification.service';
 
 @Component({
   selector: 'app-favorite-toggle',
   templateUrl: 'favorite-toggle.component.html',
   styleUrls: [],
 })
 export class FavoriteToggleComponent {
-  constructor() {}
+  constructor(private notificationService: NotificationService) {}
   @Input() isFavorite: boolean | undefined;
   @Output() onChange = new EventEmitter<boolean | undefined>();
 
   onToggle() {
     this.isFavorite = !this.isFavorite;
     this.onChange.emit(this.isFavorite);
+
+    this.notificationService.show('Info','You clicked on heart icon')
```

Comment:
Please create Enum for type of notifications

Reply:
Created Enums for type of notifications